.github/workflows/autodeploy.yaml

```
@@ -0,0 +1,26 @@
+name: Deploy Storybook to S3
+
+on:
+  push:
```

```suggestion
on:
  pull_request:
  push:
```

you can add a `pull_request:` trigger to test on this PR

---

ah nice, thanks